app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -91,6 +100,16 @@ class BrowserFragment : WebFragment(), CursorEvent {
         })
     }
 
+    override fun onEvent(event: BrowserNavigationOverlay.NavigationEvent, value: String?, autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?) {
```

Comment:
nit: one of our linters may eventually get mad at the line lengths in this function. if you move NavigationEvent to the top-level in BrowserNavigationOverlay, these imports would be shorter